udata/templates/topic/display.html

```diff
@@ -26,10 +26,10 @@
             {# Last reuses #}
             <div class="col-xs-12">
 
-                <h3>{{ _('Latest reuses') }}</h3>
+                <h3><a href="{{ url_for('topics.reuses', topic=topic) }}">{{ _('Latest reuses') }}</a></h3>
```

Comment:
Isn't this a little confusing? This won't link to the _Latest reuses_ but to the _Featured reuses_ AFAIK.

Maybe:
- rename title to _Featured reuses_
- link explicitly to _Latest reuses_

Reply:
Or remove all references to _Latest reuses_ but:
- this would mean deleting `topics.reuses` in the model
- it can be quite "dangerous" to automatically promote reuses for a topic (I think it has been (re)designed because there were some abuses)

I still like this solution 🙃

Reply:
in this context `topic.reuses` is about the route "last reuses of this topic" and not to the `reuses` property/dict of the `topic` model.

Reply:
(I won't have bothered rename the section and open a PR if there was an actual page/list about featured reuses)

Reply:
Maybe check this : https://github.com/opendatateam/udata/blob/800d0ae88438fe4a103a9930e06862a3a0952cb0/udata/core/topic/views.py#L77

Reply:
Ok for the link 🙈